genapkovl-lima.sh

```diff
@@ -209,21 +215,38 @@ if [ "${LIMA_INSTALL_IPTABLES}" == "true" ] || [ "${LIMA_INSTALL_NERDCTL}" == "t
     echo "iptables ip6tables" >> "$tmp"/etc/apk/world
 fi
 
-if [ "${LIMA_INSTALL_NERDCTL}" == "true" ]; then
+if [ "${LIMA_INSTALL_NERDCTL}" == "true" ] || [ "${LIMA_INSTALL_BUILDKIT}" == "true" ]; then
     mkdir -p "${tmp}/nerdctl"
     tar xz -C "${tmp}/nerdctl" -f /home/build/nerdctl.tar.gz
 
     mkdir -p "${tmp}/usr/local/bin/"
-    for bin in buildctl buildkitd nerdctl; do
+    bin=""
+    if [ "${LIMA_INSTALL_NERDCTL}" == "true" ]; then
+	bin="$bin nerdctl"
+    fi
+    if [ "${LIMA_INSTALL_BUILDKIT}" == "true" ]; then
+	bin="$bin buildctl buildkitd"
+    fi
+    for bin in $bin; do
         cp "${tmp}/nerdctl/bin/${bin}" "${tmp}/usr/local/bin/${bin}"
         chmod u+s "${tmp}/usr/local/bin/${bin}"
     done
     mkdir -p "${tmp}/usr/local/libexec/cni"
-    for plugin in bridge portmap firewall tuning isolation host-local; do
+    plugin="isolation"
+    if [ "${LIMA_INSTALL_CONTAINERD}" != "true" ]; then
```

Comment:
Maybe add a comment here that the remaining plugins are already installed via the cni-plugins package when containerd is installed?

Reply:
Not even sure it works when you spread them out, but maybe the config can take multiple paths ?

```
"$pkgdir"/usr/bin/containerd config default | sed "s|/opt/cni/bin|/usr/libexec/cni|g" > "$pkgdir"/etc/containerd/config.toml
}
```

```
    [plugins."io.containerd.grpc.v1.cri".cni]
      bin_dir = "/opt/cni/bin"
      conf_dir = "/etc/cni/net.d"
      conf_template = ""
      max_conf_num = 1
```

Packaging cni-isolation as an apk is another option, I suppose.

Reply:
So maybe test this before we merge?

Reply:
Could revert to installing both, it was like 10M difference anyway. Don't need so much CNI for a local VM.

Reply:
But I will try to fix it, maybe something like a symlink would do the trick short-term.

Reply:
But good idea to test it, seems that I broke `nerdctl` networking.

FATA[0004] needs CNI plugin &{"bridge" "nerdctl0" %!q(bool=true) %!q(bool=false) %!q(bool=false) %!q(bool=true) '\x00' %!q(bool=true) %!q(bool=false) '\x00' map["ranges":[[map["gateway":"10.4.0.1" "subnet":"10.4.0.0/24"]]] "routes":[map["dst":"0.0.0.0/0"]] "type":"host-local"]} to be installed in CNI_PATH ("/usr/local/libexec/cni"), see https://github.com/containernetworking/plugins/releases: exec: "/usr/local/libexec/cni/bridge": stat /usr/local/libexec/cni/bridge: no such file or directory

Reply:
Leaving both, can "optimize" it later.